content/docs/1.17/operator.md

```diff
@@ -19,29 +19,64 @@ While multiple operators might coexist watching the same set of namespaces, whic
 The Jaeger Operator version tracks one version of the Jaeger components (Query, Collector, Agent). When a new version of the Jaeger components is released, a new version of the operator will be released that understands how running instances of the previous version can be upgraded to the new version.
 {{< /info >}}
 
+## Install modes
+
+The Jaeger Operator can be installed to watch for new Jaeger resources either in the whole cluster or in specific namespaces.
+
+When watching all namespaces, use the `cluster_role.yaml` and `cluster_role_binding.yaml` files while installing the operator, and set the `WATCH_NAMESPACE` to empty in the `operator.yaml` file. With this in place, the Jaeger Operator will:
+
+- watch for events related to Jaeger resources in all namespaces
+- watch the namespaces themselves looking for the `sidecar.jaegertracing.io/inject` annotation
+- watch all deployments, to inject or remove sidecars based on the `sidecar.jaegertracing.io/inject` annotation
+- create cluster role bindings, when necessary
+
+When not using the cluster-wide resources (`ClusterRole` and `ClusterRoleBinding`), set the `WATCH_NAMESPACE` to the comma-separated list of namespaces that the Jaeger Operator should watch for events related to Jaeger resources. It is possible to have the Jaeger Operator running in a given namespace (like, `observability`) and manage Jaeger resources in another (like, `myproject`). For that, use a `RoleBinding` like the following:
```

Comment:
After reading this my impression is that I have to create `RoleBinding` per each namespace the operator is watching. Is that true?

Reply:
Yes.

Reply:
It might be better to be more explicit here and explicitly say command that for each namespace should be created a RB.

Reply:
Done!